scapy/contrib/isotp/isotp_native_socket.py

```diff
@@ -181,8 +183,8 @@ def __build_can_isotp_fc_options(self,
     # };
 
     def __build_can_isotp_ll_options(self,
-                                     mtu=CAN_ISOTP_DEFAULT_LL_MTU,
-                                     tx_dl=CAN_ISOTP_DEFAULT_LL_TX_DL,
+                                     mtu,
```

Comment:
why did you removed the default values?

Reply:
because I set them in __init__ explicitly. No need for default values?